mautic/mautic
# Fix datetime import

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Noticed date time is imported as UTC time zone. Then displayed in contact profile in time zone related to configuration.  User expect see date time same in contact profile like was imported.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set timezone in configuration to Europe/Paris
3. Create custom date time field
4. Run this simple import
[import.zip](https://github.com/mautic/mautic/files/6220224/import.zip)
5. Based on import date time value `2020-02-02 21:49:59` expect same you should see after import in contact profile. But you see `2020-02-02 22:49:59`
6. After PR you should see after import `2020-02-02 21:49:59`. Same for both kind of imports: UI and background




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9854"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
